package.json

```diff
@@ -18,7 +18,7 @@
   },
   "scripts": {
     "tsc:blade": "yarn workspace @razorpay/blade typecheck",
-    "lint": "eslint --ext js,jsx,ts,tsx packages && stylelint '**/*.{js,jsx,ts,tsx}'",
+    "lint": "eslint --ext js,jsx,ts,tsx packages/blade --fix && stylelint '**/*.{js,jsx,ts,tsx}'",
```

Comment:
intended? maybe create a new script with `lint:blade` name?

Reply:
No, not intended.
```suggestion
    "lint": "eslint --ext js,jsx,ts,tsx packages && stylelint '**/*.{js,jsx,ts,tsx}'",
```